libnymea-zigbee/zcl/zigbeecluster.cpp

```diff
@@ -365,6 +365,9 @@ bool ZigbeeCluster::verifyNetworkError(ZigbeeClusterReply *zclReply, ZigbeeNetwo
         // The request has been transported successfully to he destination, now
         // wait for the expected indication or check if we already recieved it
         zclReply->m_apsConfirmReceived = true;
+        if (!zclReply->m_apsConfirmReceived) {
+            zclReply->m_timeoutTimer.start();
```

Comment:
you set it to true and then you check if is false?

Reply:
right. fixed, thanks